engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/mutable/NewNode.java

```diff
@@ -1,4 +1,4 @@
-package org.enso.interpreter.node.expression.builtin.array;
+package org.enso.interpreter.node.expression.builtin.mutable;
```

Comment:
I still really think these `NewNode` nodes should be done using in-language variadics. :P